### PR TITLE
K8SPXC-370 Fix 'scheduled-backup' test

### PR DESCRIPTION
**Pull request by hors**
[K8SPXC-370](https://jira.percona.com/browse/K8SPXC-370)

    * fix 'scheduled-backup' test
    * start to use new helm repo for minio charts

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

File: pkg/pxc/backup/scheduled.go [hunk 1]
```diff
@@ -102,7 +102,7 @@ func (bcp *Backup) scheduledJob(spec *api.PXCScheduledBackupSchedule, strg *api.
 									kind: PerconaXtraDBClusterBackup
 									metadata:
 									  namespace: "` + bcp.namespace + `"
-									  name: "cron-${pxcCluster:0:16}-$(date -u "+%Y%m%d%H%M%S")-${suffix}"
+									  name: "cron-${pxcCluster:0:16}-$(date -u "+%Y%m%d%H%M%S")-` + spec.StorageName + `-${suffix}"
```
Comment (delgod) [inline pkg/pxc/backup/scheduled.go:105]: should we limit the storage name to 16 chars?

**Review by delgod — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| haproxy | passed |
| recreate | passed |
| scaling | passed |
| big-data | passed |
| storage | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| init-deploy | passed |
| restore-to-encrypted-cluster | failed |
| upgrade | passed |
| self-healing-advanced | passed |
| limits | passed |
| upgrade-consistency | passed |
| security-context | failed |
| operator-self-healing | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | failed |
| demand-backup | failed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| scheduled-backup | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| big-data | passed |
| haproxy | passed |
| storage | passed |
| scaling | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| init-deploy | passed |
| restore-to-encrypted-cluster | passed |
| upgrade | passed |
| limits | passed |
| upgrade-consistency | passed |
| self-healing-advanced | failed |
| security-context | passed |
| monitoring | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| scheduled-backup | failed |
| one-pod | passed |
| auto-tuning | passed |
| operator-self-healing | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| self-healing | passed |
| self-healing-advanced | passed |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| scaling | passed |
| scheduled-backup | failed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |
| haproxy | passed |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| big-data | passed |
| operator-self-healing | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | passed |
| haproxy | passed |
| storage | passed |
| recreate | passed |
| scaling | passed |
| self-healing | passed |
| init-deploy | passed |
| scaling-proxysql | passed |
| restore-to-encrypted-cluster | passed |
| upgrade | passed |
| limits | passed |
| self-healing-advanced | passed |
| upgrade-consistency | passed |
| monitoring | passed |
| security-context | passed |
| operator-self-healing | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| scheduled-backup | failed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | passed |
| storage | passed |
| recreate | passed |
| scaling | passed |
| haproxy | passed |
| self-healing | passed |
| init-deploy | passed |
| scaling-proxysql | passed |
| restore-to-encrypted-cluster | passed |
| upgrade | passed |
| limits | passed |
| self-healing-advanced | passed |
| upgrade-consistency | passed |
| monitoring | passed |
| security-context | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| operator-self-healing | passed |
| one-pod | passed |
| scheduled-backup | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| storage | passed |
| recreate | passed |
| haproxy | passed |
| big-data | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| init-deploy | passed |
| upgrade | passed |
| restore-to-encrypted-cluster | passed |
| limits | passed |
| self-healing-advanced | passed |
| upgrade-consistency | passed |
| security-context | passed |
| monitoring | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| operator-self-healing | passed |
| affinity | passed |
| one-pod | passed |
| scheduled-backup | failed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| big-data | passed |
| scaling | passed |
| haproxy | passed |
| recreate | passed |
| init-deploy | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| restore-to-encrypted-cluster | passed |
| upgrade | passed |
| limits | passed |
| self-healing-advanced | passed |
| upgrade-consistency | passed |
| monitoring | passed |
| security-context | passed |
| operator-self-healing | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| scheduled-backup | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| haproxy | passed |
| demand-backup-encrypted-with-tls | failed |
| init-deploy | passed |
| limits | passed |
| big-data | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| scaling | passed |
| auto-tuning | passed |
| scaling-proxysql | passed |
| proxysql-sidecar-res-limits | passed |
| upgrade | passed |
| users | passed |
| upgrade-consistency | passed |
| security-context | passed |
| storage | passed |
| self-healing | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |

File: pkg/pxc/backup/scheduled.go [hunk 2]
```diff
@@ -102,7 +102,7 @@ func (bcp *Backup) scheduledJob(spec *api.PXCScheduledBackupSchedule, strg *api.
 									kind: PerconaXtraDBClusterBackup
 									metadata:
 									  namespace: "` + bcp.namespace + `"
-									  name: "cron-${pxcCluster:0:16}-$(date -u "+%Y%m%d%H%M%S")-${suffix}"
+									  name: "cron-${pxcCluster:0:16}-$(date -u "+%Y%m%d%H%M%S")-` + trimNameRight(spec.StorageName, 16) + `-${suffix}"
```
Comment (delgod) [inline pkg/pxc/backup/scheduled.go:105]:
```suggestion
									  name: "cron-${pxcCluster:0:16}-` + trimNameRight(spec.StorageName, 16) + `-$(date -u "+%Y%m%d%H%M%S")-${suffix}"
```

**Review by delgod — CHANGES_REQUESTED** [review]

File: pkg/pxc/backup/scheduled.go (hunk 2 above)
Comment (hors) [reply pkg/pxc/backup/scheduled.go]: done

**Review by delgod — APPROVED** [review]

**Review by delgod — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| recreate | passed |
| haproxy | passed |
| big-data | passed |
| storage | passed |
| scaling-proxysql | passed |
| init-deploy | passed |
| self-healing | passed |
| upgrade | passed |
| restore-to-encrypted-cluster | failed |
| limits | passed |
| upgrade-consistency | passed |
| security-context | failed |
| monitoring | passed |
| demand-backup | failed |
| monitoring-2-0 | passed |
| self-healing-advanced | passed |
| affinity | passed |
| scheduled-backup | failed |
| one-pod | failed |
| operator-self-healing | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup-encrypted-with-tls | failed |

**Pull request merged**